user-guide/Advanced_Modules/Automation_module/Running_Automation_scripts.md

```diff
@@ -11,3 +11,5 @@ Once a script has been designed, it can be executed in several ways:
 - With DataMiner Scheduler: scripts can be run automatically at certain predefined points in time. See [Scheduler](xref:scheduler).
 
 - With DataMiner Correlation: scripts can be run automatically whenever user-defined events occur. See [Running an Automation script from a Correlation rule](xref:Running_an_Automation_script_from_a_Correlation_rule).
+
+- You can make an API of your script using [User-defined APIs](xref:UD_APIs). How to make your script into an API is explained on the [Defining a new API page](xref:UD_APIs_Define_New_API#2-creating-the-apidefinitions).
```

Comment:
Is this sentence ok? "How to make your script into an API..."

Reply:
```suggestion
- You can make an API of your script using [User-defined APIs](xref:UD_APIs). How to make an API from your automation script is explained on the [Defining a new API page](xref:UD_APIs_Define_New_API#2-creating-the-apidefinitions).
```
This better? @Pieter-JanV

Reply:
Strictly speaking, this is correct, but I'd recommend simplifying it a bit:

```suggestion
- You can make an API of your script using [User-defined APIs](xref:UD_APIs). For more information, see [Defining a new API page](xref:UD_APIs_Define_New_API#2-creating-the-apidefinitions).
```